lib/portage/update.py

```diff
@@ -429,11 +425,13 @@ def match_callback(repo_name, atoma, atomb):
                             # add a comment with the update command, so
                             # the user can clearly see what happened
                             contents[pos] = "# %s\n" % " ".join(
```

Comment:
Here's another missed `.format` opportunity.

Reply:
These 3 should be fixed in a new commit